### ISSUE TITLE
loadedGesture is null when CloseGesture() is called

### ISSUE THREAD

**Issue by Dremorbp** [carlfranklin/GesturePak2V1#1]
When running GesturePakRecorder and recording a gesture, after the countdown has ended, the program stops and gives a null value from the loadedGesture object inside the CloseGesture() method when loadedGesture.SaveFile() is called, but the XML file is saved sucessfully. 

Is there any known bug regarding this, or am I missing something?

![gesturepakrecorder](https://cloud.githubusercontent.com/assets/7127122/15361463/849939ae-1d09-11e6-9e3d-5ef0e992bac5.PNG)

**Comment by carlfranklin** [comment]
First I've heard of it. Fails every time??

**Comment by Dremorbp** [comment]
Yes, don't know if it's anything related to the saved XML directory containing special characters, but I will try to run it from a diferent computer and see what happens. Do you have any idea or suggestion why this is happening?

Thank you.

**Comment by carlfranklin** [comment]
I had an issue with an earlier version running on non-English Windows but I
addressed the issue. Perhaps it's a new bug.
